lib/screen/bloc/router/router_bloc.dart

```diff
@@ -175,13 +175,18 @@ class RouterBloc extends AuBloc<RouterEvent, RouterState> {
             .initIfDefaultAccount());
         await migrationUtil.migrateIfNeeded();
         try {
+          final primaryAddressInfo =
+              await _addressService.getPrimaryAddressInfo();
```

Comment:
consider checking not only if the primary address is in keychain, but also user have the wallet storage object of that address

Reply:
we should make sure that always have the wallet storage of primary address. It will make the flow more simple.